cluster-provision/README.md

```diff
@@ -67,7 +67,7 @@ gocli provision okd \
 --installer-pull-token-file <installer_pull_token_file> \
 --installer-repo-tag release-4.1 \
 --installer-release-image quay.io/openshift-release-dev/ocp-release:4.1.0-rc.7 \
-kubevirtci/okd-base@sha256:caf1c3f63dc5f3137795084e492a48c802a7ef2e7f7fbe1d67e5cff684d376a3
+kubevirtci/okd-base@sha256:1e21a7b0fc959ae2a47d1d6462ceb74f9e9181d52f9bc618e61bcab80bedaa9e
```

Comment:
oh, this is incorrect, also in the old version. Can you set it to `259e776998da3a503a30fdf935b29102443b24ca4ea095c9478c37e994e242bb` please, see line 38.